docs/source/running-autometa.rst

```diff
@@ -30,21 +24,21 @@ function.
 .. code-block:: python
     :linenos:
```

Comment:
Do we need line numbers? It just looks pretty different from rest of the code.

Reply:
No, we do not need line numbers. This is now removed.